src/components/Databoard-review.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from "react";
+// import getData from "../getData";
+import ResultSegment from "./ResultSegment";
+
+export default function Databoard({ timekey }) {
+  // set state variables to store data to be displayed
+  const [dataLoaded, setDataLoaded] = useState(false);
+
+  const [homePossessionData, setHomePossessionData] = useState([]);
+  const [awayPossessionData, setAwayPossessionData] = useState([]);
+
+  const [homeShotData, setHomeShotData] = useState([]);
+  const [awayShotData, setAwayShotData] = useState([]);
+
+  const [homeShotOnTargetData, setHomeShotOnTargetData] = useState([]);
+  const [awayShotOnTargetData, setAwayShotOnTargetData] = useState([]);
+
+  const [homeCornersData, setHomeCornersData] = useState([]);
+  const [awayCornersData, setAwayCornersData] = useState([]);
+
+  useEffect(() => {
+    console.log("Fetching data");
+    fetch(process.env.REACT_APP_API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        let matchData = data.match;
+        let liveData = matchData.liveData;
+        let lineupData = liveData.lineups;
+        let homeData = lineupData.home.stats;
+        let awayData = lineupData.away.stats;
+
+        setHomePossessionData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+        setAwayPossessionData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotOnTargetData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotOnTargetData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeCornersData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
+        setAwayCornersData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
```

Comment:
all these functions are doing something very similar, if you're in a position where you have code which looks to be doing the same thing multiple times, I would think about how this could be refactored to remove some of the repetition. For example here each function is using an `array` and `type` to return a filtered array, could you make a helper function to reduce some of this repetition?